operator/cmd/mesh/root.go

```diff
@@ -47,12 +47,14 @@ If set to true, the user is not prompted and a Yes response is assumed in all ca
 This flag can be specified multiple times to overlay multiple files. Multiple files are overlaid in left to right order.`
 	installationCompleteStr  = `Installation complete`
 	ForceFlagHelpStr         = `Proceed even with validation errors.`
+	verificationHelpStr      = `Verify after installation`
 	KubeConfigFlagHelpStr    = `Path to kube config.`
 	ContextFlagHelpStr       = `The name of the kubeconfig context to use.`
 	HubFlagHelpStr           = `The hub for the operator controller image.`
 	TagFlagHelpStr           = `The tag for the operator controller image.`
 	OperatorNamespaceHelpstr = `The namespace the operator controller is installed into.`
 	ComponentFlagHelpStr     = "Specify which component to generate manifests for."
+	VerifyCRInstallHelpStr   = "Verify after installation"
```

Comment:
strings are duplicated, do we need both?

Reply:
No. I'll keep this one and remove the other.